x-pack/plugins/security_solution/server/lib/app_features/security_kibana_sub_features.ts

```diff
@@ -396,14 +396,18 @@ const hostIsolationSubFeature: SubFeatureConfig = {
       groupType: 'mutually_exclusive',
       privileges: [
         {
-          api: [`${APP_ID}-writeHostIsolationRelease`],
           id: 'host_isolation_all',
           includeIn: 'none',
           name: 'All',
           savedObject: {
             all: [],
             read: [],
           },
+          // FYI: The current set of values below (`api`, `ui`) cover only `release` response action.
```

Comment:
I'm hopping that tidbits of info like this can help along understanding how the framework around loading different privileges into existing Sub-Features